crates/cli-support/src/js/mod.rs

```diff
@@ -269,6 +270,40 @@ impl<'a> Context<'a> {
         reset_indentation(&shim)
     }
 
+    // generates somthing like
+    // ```js
+    // const imports = {
+    //   __wbindgen_placeholder__: {
+    //     __wbindgen_throw: function(..) { .. },
+    //     ..
+    //   }
+    // }
+    // ```
+    fn generate_deno_imports(&self) -> String {
+        let mut imports = "const imports = {\n".to_string();
+        imports.push_str(&format!("  {}: {{\n", crate::PLACEHOLDER_MODULE));
+
+        for (id, js) in crate::sorted_iter(&self.wasm_import_definitions) {
+            let import = self.module.imports.get(*id);
+            imports.push_str(&format!("{}: {},\n", &import.name, js.trim()));
+        }
+
+        imports.push_str("\t}\n};\n\n");
+
+        imports
+    }
+
+    fn generate_deno_wasm_loading(&self, module_name: &str) -> String {
+        format!(
+            "const file = new URL(import.meta.url).pathname;
+            const wasmFile = file.substring(0, file.lastIndexOf(Deno.build.os === 'windows' ? '\\\\' : '/') + 1) + '{}_bg.wasm';
+            const wasmModule = new WebAssembly.Module(Deno.readFileSync(wasmFile));
```

Comment:
It looks like deno support for native imports was removed in https://github.com/denoland/deno/pull/5135, and perhaps https://github.com/denoland/deno/issues/5609 is a tracking issue for that returning? Mind leaving a comment referring to that issue here in case we need to check up on it in the future?